### PR TITLE
Remove duplicate links

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: docs/framework/data/adonet/sql/linq/sequence-operators.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ Generally speaking, [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-
 |-----------------------------|  
 |<xref:System.Linq.Enumerable.Where%60%601%28System.Collections.Generic.IEnumerable%7B%60%600%7D%2CSystem.Func%7B%60%600%2CSystem.Int32%2CSystem.Boolean%7D%29?displayProperty=nameWithType>|  
 |<xref:System.Linq.Enumerable.Select%60%602%28System.Collections.Generic.IEnumerable%7B%60%600%7D%2CSystem.Func%7B%60%600%2C%60%601%7D%29?displayProperty=nameWithType>|  
-|<xref:System.Linq.Enumerable.Select%60%602%28System.Collections.Generic.IEnumerable%7B%60%600%7D%2CSystem.Func%7B%60%600%2C%60%601%7D%29?displayProperty=nameWithType>|
```
Comment (rpetrusha) [inline docs/framework/data/adonet/sql/linq/sequence-operators.md]: The intent here was to link to a different the overload: https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.select?view=netframework-4.8#System_Linq_Enumerable_Select__2_System_Collections_Generic_IEnumerable___0__System_Func___0_System_Int32___1__

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for noticing and addressing the duplication, @NextTurn. The intent, though, was to link to the other overload of `Enumerable.Select`.

File: docs/framework/data/adonet/sql/linq/sequence-operators.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/data/adonet/sql/linq/sequence-operators.md]: You can see the xref format here:
https://xref.docs.microsoft.com/autocomplete?text=Enumerable.Select``2

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you, @NextTurn, for making the additional change. I'll merge your PR now.

**Pull request merged**